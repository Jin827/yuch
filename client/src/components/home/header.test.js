import React from 'react';
import { shallow } from 'enzyme';
import Header from './header';

describe('<Header />', () => {
  const wrapper = shallow(<Header />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('h2').text()).toBe(
      'NO MSG!오늘도 열심히 일한 당신에게 당신만을 위한 착한 가격의 집밥을선물하세요.',
    );
  });

  it('should contain Nav component', () => {
    expect(wrapper.find('withRouter(Connect(Nav))').exists()).toBe(true);
  });
});
