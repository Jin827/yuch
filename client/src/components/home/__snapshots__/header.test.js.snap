// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header>
  <withRouter(Connect(Nav)) />
  <div
    className="header-text--box"
  >
    <h2>
      <span
        className="point1 f-en lh-3"
      >
        NO MSG!
      </span>
      <br />
      오늘도 열심히 일한 당신에게 
      <br />
      당신만을 위한 
      <span
        className="point2"
      >
        착한 가격의 집밥을
      </span>
      선물하세요.
    </h2>
  </div>
</header>
`;
